tests/agent_equipment/test_canonical.py

```
@@ -0,0 +1,381 @@
+                        version=version,
+                    ),
+                    self.assertRaises(ValueError),
+                ):
+                    fixture_manifest(
+                        root=root,
+                        runtime=runtime,
+                        implementation_name=implementation_name,
+                        version=version,
+                    )
+
+            with self.assertRaises(ValueError):
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)